cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
The `include` function is using the `namespace` argument. Ensure that the included URL configurations in `cinema.urls` and `user.urls` have an `app_name` defined to support namespacing. If not, this will cause an error.